internal/providers/amazon/cloudformation.go

```diff
@@ -0,0 +1,98 @@
+// Copyright © 2019 Banzai Cloud
```

Comment:
I think this should be under `pkg/providers/amazon` because it is not pipeline specific, it depends only on built-in, 3rd party, and pkg packages.